tools/include/pass.hpp

```diff
@@ -81,16 +81,19 @@ void module_pass_manager_apply(const T& x, module_pass_manager& mpm)
 
 } // namespace detail
 
-<%
-interface('pass',
-    virtual('name', returns='std::string', const=True),
-    virtual('apply', returns='void', mpm='module_pass_manager &', const=True, default='migraphx::detail::module_pass_manager_apply'),
-    virtual('apply', returns='void', p='program &', const=True, default='migraphx::nop')
-)
-%>
+<%interface(
+    'pass',
+    virtual('name', returns = 'std::string', const = True),
+    virtual('apply',
+            returns = 'void',
+            mpm     = 'module_pass_manager &',
+            const   = True,
+            default = 'migraphx::detail::module_pass_manager_apply'),
+    virtual('apply', returns = 'void', p = 'program &', const = True, default = 'migraphx::nop'))%>
 
 /// Used in the targets to enable/disable compiler passes
-pass enable_pass(bool enabled, pass p);
+MIGRAPHX_EXPORT pass
+enable_pass(bool enabled, pass p);
```

Comment:
Make this line look identical to the one from src/include/migrpahx/pass.hpp

Reply:
Thanks for suggestion, I changed it.